src/apps/default.ts

```diff
@@ -1,8 +1,8 @@
-import type { ApplicationFunctionOptions, Probot } from "../index";
-import { loadPackageJson } from "../helpers/load-package-json";
+import type { ApplicationFunctionOptions, Probot } from "../index.js";
```

Comment:
wait shouldn't these be `.ts` extensions?

Reply:
This is a very weird behaviour for TypeScript, where you need to add the extensions it has in the transpiled source code, which is `.js`.

This is even true for `import type` for some reason. If your code is ESM, which would be one line to change in package.json, having no extension or `.ts`, even in an `import type` will throw a compile error.

Reply:
There is the [`allowImportingTsExtensions` option](https://www.typescriptlang.org/tsconfig#allowImportingTsExtensions) that does allow importing `.ts` extension, but it doesn't work when you use typescript as you compiler for ts -> js